modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -0,0 +1,71 @@
+[id="cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_{context}"]
+= Configuration of JSON log data for default Elasticsearch
+
+With the default Elasticsearch log store, you must create an index for each format if the JSON log entries _have different formats_.
+
+[IMPORTANT]
+====
+You must create a separate index for each different JSON log format. Otherwise, forwarding different formats to the same index can cause type conflicts and cardinality problems.
+====
+
+To provide a different index for each format, you configure the `ClusterLogForwarder` custom resource. You use a structure type to construct the index name from a category, Kubernetes label, or input selector.
+
+.Structure types
+
+The `ClusterLogForwarder` provides the following structure types for use with the default Elasticsearch log store:
+
+* `structuredTypeName`: (string, optional) If `structuredTypeKey` is not set or its key is not present, use the value of `structuredTypeName` as the structured type.
+* `structuredTypeKey`: (string, optional) If either of the following meta-data keys is set and its value is non-empty, use the value of the key as the structured type.
+** `kubernetes.labels.<key>`: Use the string value of the `kubernetes` label that has the specified `<key>`.
```

Comment:
Use the string value of the `kubernetes` label that has the specified `<key>` in pod

Reply:
Updated. Thank you.